src/sentry/models/group.py

```diff
@@ -158,24 +156,13 @@ def from_event_id(self, project, event_id):
         Resolves the 32 character event_id string into
         a Group for which it is found.
         """
-        from sentry.models import EventMapping, Event
+        from sentry.models import SnubaEvent
         group_id = None
 
-        # Look up event_id in both Event and EventMapping,
-        # and bail when it matches one of them, prioritizing
-        # Event since it contains more history.
-        for model in Event, EventMapping:
```

Comment:
I don't think we can remove the EventMapping check yet.
EventMapping is saved if the event is sampled (see https://github.com/getsentry/sentry/blob/618bb0cc6f8edd36d0c6462e25f496f67ba3840f/src/sentry/event_manager.py#L717-L731)

Even after removing events from postgres, eventmapping is going to stay.

Reply:
Same as below, won't all the events be in Snuba?